### DISCUSSION TITLE
Project copied from one mac to another fails to play clips

### DISCUSSION THREAD

**Discussion by nigelhaslam** [pietrop/digital-paper-edit-electron#76] [Q&A] [answered]
Hi,
I'm working with a team and have recently shifted to MAC platform where I'm finding Autoedit3 1.6.0 more stable.
I am trying to copy the entire digital-paper-edit-electron folder from the machine where the clips were ingested, to another mac so that we can work separately without having to re transcribe.
Unfortunately as mentioned in the subject line, the clips won't play back on the other machine.

Source mac was a recent Macbook Pro - destination is an iMac
Do you have any suggestions?

**Comment by pietrop** [comment, marked as answer]
Hi @motioncircus ,
Yes, I think I know what's happening. 

The file path reference to the proxies in the transcription data, has got a reference to the home folder in it's path, so when you change computer unless that's changed as well, it wouldn't be able to find it, if that makes sense?

ok, let's break this down. 


As you know there's a folder in the user systems. on a Mac it's 

```diff
+/Users/${home_folder}/Library/Application\ Support/digital-paper-edit-electron
```

This is what you'd copy from one mac to another to move your whole projects across. 

This contains, the proxies inside the `/media` folder, with the previews.

And there's a `/db`  folder that contains the transcription data.

```
annotations.json
labels.json
paperedits.json
projects.json
transcripts.json
```


Out of these once `transcripts.json`

Here's an example

```diff
[
    {
        "projectId": "366724cd3aab487b9e3b4ed3be2b3284",
        "title": "Transcription Name",
        "description": "",
+        "path": "/Users/${home_folder}/Movies/Transcription Name-g3Af2ntyQyU.mp4",
+        "url": "/Users/${home_folder}/Library/Application Support/digital-paper-edit-electron/media/Transcription Name-g3Af2ntyQyU.c797b714ce5d4283b7f778c3ce79e56a.mp4",
        "status": "done",
        "_id": "c797b714ce5d4283b7f778c3ce79e56a",
        "id": "c797b714ce5d4283b7f778c3ce79e56a",
        "metadata": {
+            "filePathName": "/Users/${home_folder}/Movies/Transcription Name-g3Af2ntyQyU.mp4",
            "fileName": "Transcription Name-g3Af2ntyQyU.mp4",
            "date": "2020-04-29 20:07:42",
            "reelName": "NA",
            "timecode": "NA",
            "r_frame_rate": "30000/1001",
            "fps": 29.97,
            "duration": 2894.0912,
            "sampleRate": 44100
        },
+        "videoUrl": "/Users/${home_folder}/Library/Application Support/digital-paper-edit-electron/media/Transcription Name-g3Af2ntyQyU.c797b714ce5d4283b7f778c3ce79e56a.mp4",
        "transcript": {
            "words": [
                 {
                    "id": 0,
                    "text": "Good",
                    "start": 4.1,
                    "end": 5.1
                },
                ...
            ],
            "paragraphs": [
                {
                    "start": 4.1,
                    "end": 7.22,
                    "speaker": "U_UKN"
                },
                ...
            ],
        },
        "clipName": "Transcription Name-g3Af2ntyQyU.mp4",
        "sttEngine": "AssemblyAI",
+        "audioUrl": "/Users/${home_folder}/Library/Application Support/digital-paper-edit-electron/media/Transcription Name-g3Af2ntyQyU.c797b714ce5d4283b7f778c3ce79e56a.wav"
    },
    ...
 ]
````

As you can see above I've highlighted in green the relevant part, and replace my home directory name with `${home_folder}`.

A possible solution is to

1. on the new computer
2. Close autoEdit
3. Open `transcripts.json` and do a find and replace for you home folder name, with the one of the new computer.
4. save
5. open autoEdit

See if this work


As a note, to handle `.json` files, you can use [Visual studio code](https://code.visualstudio.com/) or something equivalent to help with the formatting.
